confluence-copy-link-buttons.user.js

```diff
@@ -7,7 +7,7 @@
 // @license      MIT
 // @homepageURL  https://github.com/rybak/atlassian-tweaks
 // @include      https://confluence*
-// @match        https://confluence.example.com
+// @match        https://confluence.example.com/*
```

Comment:
This change is not just a Violentmonkey fix, it's a fix for users in general – our instructions for custom domains tell how to copy-paste the original `@match`, but, of course, it won't work without `/*` on the end. Thanks.

For reference, the correct `@match` entry is used in [confluence-space-avatar-favicons.user.js](https://github.com/rybak/atlassian-tweaks/blob/main/confluence-space-avatar-favicons.user.js#L7-L8).

Reply:
Merged to `main` in commit [f866d49](https://github.com/rybak/atlassian-tweaks/commit/f866d496ff523ebde01156732f3fa10d5b64a8d2) (confluence-copy-link-buttons: fix match entry for example.com, 2025-05-06).